### DISCUSSION TITLE
Github Desktop Issue

### DISCUSSION THREAD

**Discussion by prdai** [community/community#16135] [Other Feature Feedback, Questions, & Ideas] [answered]
![Issue Image](https://user-images.githubusercontent.com/79456372/167094889-c16fdac0-0d84-4a29-98f8-2346ac86ea0e.png)
When I commit something from the GitHub desktop.
There is another user that is created in the commit.
So I always need to co-auth my user name.
And I checked the user name it's a 404.
With best regards,
Ranuga

**Comment by queenofcorgis** [comment]
Hi there! Is this on anything you create across repositories or have you forked a repository from another author?

**Reply by prdai** [reply]
No. It's my personal repos.
I don't use templates or anything.
Best regards

**Comment by steveward** [comment, marked as answer]
Can you go to `File` > `Options` > `Git` and verify that the email address listed there matches an email address associated with your currently logged-in GitHub account? You can also select an email from your account from the dropdown.

**Reply by prdai** [reply]
Cannot push these commits as they contain an email address marked as private on GitHub. To push anyway, visit https://github.com/settings/emails, uncheck "Keep my email address private", then switch back to GitHub Desktop to push your commits. You can then enable the setting again.

**Reply by prdai** [reply]
I get the above error

**Reply by prdai** [reply]
The profile picture changed

**Comment by pentr32** [comment]
Go to Start > Credentials Manager > Windows Credentials > and remove the account that you don't want.
